packages/graphqlsp/src/index.ts

```diff
@@ -62,6 +62,11 @@ function create(info: ts.server.PluginCreateInfo) {
     const originalDiagnostics =
       info.languageService.getSemanticDiagnostics(filename);
 
+    const hasGraphQLDiagnostics = originalDiagnostics.some(x =>
+      ALL_DIAGNOSTICS.includes(x.code)
```

Comment:
Could be a bit strict here and see whether the length of both arrays is the same